.github/workflows/test.yml

```
@@ -0,0 +1,126 @@
+          emacs -batch -L . -L test \
+            -l ~/.emacs.d/init.el \
+            --eval "(require 'undercover)" \
+            --eval "(undercover \"*.el\" (:exclude \"test/*.el\"))" \
+            --eval "(require 'ert)" \
+            -l test/bfepm-test.el \
+            -l test/bfepm-config-test.el \
+            -l test/bfepm-utils-test.el \
+            -f ert-run-tests-batch-and-exit
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)